packages/common/src/core/types/IHubProject.ts

```diff
@@ -35,3 +29,13 @@ export type IHubProjectEditor = IHubItemEntityEditor<IHubProject> & {
   // Groups is an ephemeral property, so we prefix with _
   _groups?: string[];
 };
+
+// *** THIS IS DEPRECATED ***
+// This is deprecated and will be removed during our next planned
+// breaking change. Please use HubEntityStatus instead
+export enum PROJECT_STATUSES {
```

Comment:
Change this of a jsdoc style comment and add `@internal` to the start so it's kept out of the docs